extension/spark-doris-connector/src/main/java/org/apache/doris/spark/cfg/ConfigurationOptions.java

```diff
@@ -66,11 +66,9 @@
 
     String DORIS_WRITE_FIELDS = "doris.write.fields";
 
-    String SINK_BATCH_SIZE = "sink.batch.size";
     String DORIS_SINK_BATCH_SIZE = "doris.sink.batch.size";
-    int SINK_BATCH_SIZE_DEFAULT = 1024;
+    int SINK_BATCH_SIZE_DEFAULT = 10000;
 
-    String SINK_MAX_RETRIES = "sink.max-retries";
     String DORIS_SINK_MAX_RETRIES = "doris.sink.max-retries";
     int SINK_MAX_RETRIES_DEFAULT = 3;
```

Comment:
Set `SINK_MAX_RETRIES_DEFAULT` to 1, same as flink connector.
And we also need to modify `DorisStreamLoadSink.scala`, because when `SINK_MAX_RETRIES_DEFAULT` is 1, there is no retry at all. We should keep it consistent with flink connector.

Reply:
ok